conf/etc/php/7.2/fpm/php.ini

```diff
@@ -1041,7 +1041,7 @@ smtp_port = 25
 
 ; For Unix only.  You may supply arguments as well (default: "sendmail -t -i").
 ; http://php.net/sendmail-path
-;sendmail_path =
+sendmail_path = "/usr/sbin/ssmtp -t"
```

Comment:
It's not a path if it's got a -t in it, then it's a command.

Reply:
Take it up with PHP 😛